storybooknative/src/InputField/index.js

```diff
@@ -90,6 +104,12 @@ StyledInlineLabel.defaultProps = {
   theme: defaultTokens,
 };
 
+const StyledInputWrapper = styled(View, () => ({
+  width: "100%",
+  zIndex: 2,
```

Comment:
I believe there are some specific values for zIndex in orbit-design-tokens.

Reply:
@denbalogh Could you please have a look at this?

Reply:
@wEprous Should there be more z-index tokens for this use case or we stick with the custom value 2 that you use in the web version?

Reply:
Stick to the same `z-index` as we use in React. We still didn't decide how to approach to z-indexes and the values of `1` or `2` are good, it's just relative to a parent 😉